docs/FAQ.md

```diff
@@ -5,17 +5,76 @@ nav_order: 11
 ---
 # Frequently Asked Questions
 
-### Why does `explain()` show that the GPU will be used even after setting `spark.rapids.sql.enabled` to `false`?
+* TOC
+{:toc}
```

Comment:
Note that this causes an undesirable artifact when rendering with Markdown.  See https://github.com/NVIDIA/spark-rapids/blob/db7831a32c57df8cd91dbae7d88f49523401d7a8/docs/FAQ.md

Reply:
I personally think that is fine.  We already have issue with a table at the beginning of each markdown file so I am not too concerned.